src/main/java/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package service.impl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import service.ParserService;
+
+public class ParserServiceImpl implements ParserService {
+    public static final int ACTIVITY_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public Map<String, Integer> parseData(List<String> dataFromFile) {
```

Comment:
create your own class to store fruit data
```suggestion
    public List<FruitTransaction> parseData(List<String> dataFromFile) {
```